samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -29,7 +29,7 @@ Features
 
 This sample implements or demonstrates the following features:
 
-* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem <nrfxlib:nrf_modem>` and :ref:`nrf_cloud <lib_nrf_cloud>` libraries.
+* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem` and :ref:`lib_nrf_cloud` library.
```

Comment:
```suggestion
* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem` and :ref:`lib_nrf_cloud` libraries.
```
This sentence is referring to two libraries, so the word should be plural.

Reply:
@plskeggs please see the output here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-10344/nrf/samples/nrf9160/nrf_cloud_mqtt_multi_service/README.html#features. :ref:\`nrf_modem\` already expands to Modem library.

Reply:
I see.  Thanks.